server.py

```diff
@@ -550,6 +554,42 @@ def get(self):
 
         return return_dict
 
+class PendingMergesResource(restful.Resource):
+    def get(self):
+        pass
+
+    def put(self, region):
```

Comment:
ugh, when i originally wrote this, i had it as POST, then i thought i was wrong, and switched it to PUT T_T

yeah its post now